applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -137,7 +137,8 @@ class MapperVertexMorphing : public Mapper
             Initialize();
 
         BuiltinTimer timer;
-        KRATOS_INFO("\nShapeOptimization") << "Starting mapping of " << rOriginVariable.Name() << "..." << std::endl;
```

Comment:
this doesn't work?

Reply:
It does, but if there is a time stamp in front the output looks strange:
```
11:26:52 - 
ShapeOpt: Optimization problem converged within a relative objective tolerance of  1.0 %. 
```